pkg/loadbalancer/experimental/writer.go

```diff
@@ -432,15 +451,16 @@ func (w *Writer) updateBackends(txn WriteTxn, serviceName loadbalancer.ServiceNa
 			be.ZoneID = bep.ZoneID
 		}
 
-		be.Instances = be.Instances.Set(
-			serviceName,
+		be.ShadowInstances = be.ShadowInstances.Set(ServiceNameAndSource{serviceName, source},
```

Comment:
This adds it to both `be.Instances` and `be.ShadowInstances`, it should be only in one or the other so we don't waste space (the TableRow output as seen in clusterip.txtar is also confusing if it's in both).

Reply:
> This adds it to both be.Instances and be.ShadowInstances, it should be only in one or the other so we don't waste space

This is rewritten as discussed on Slack, now there is just a single field inside the Backend type for instances.

> the TableRow output as seen in clusterip.txtar is also confusing if it's in both

Corrected.